app/src/main/java/unipd/se18/ocrcamera/TestElement.java

```diff
@@ -0,0 +1,91 @@
+package unipd.se18.ocrcamera;
+
+import android.graphics.Bitmap;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * Class that contains a single test element, used in PhotoTester to build a single test and on AdapterTestElement to show data in a listview
+ * @author Luca Moroldo, Francesco Pham
+ */
+
+public class TestElement {
+
+    private Bitmap picture;
+    private JSONObject jsonObject;
```

Comment:
I did not know the JSONObject object, I did a search on the internet and it seems very useful.
A little advice for the future: if you use new objects, add the link where the functions of the objects are explained, to simplify the review.

Reply:
thank you for your advice. JSONObject is available in the standard JDK so there are loads of documentations available on the web